crates/nu-command/src/filters/select.rs

```diff
@@ -140,6 +165,16 @@ fn select(
                 }
             }
 
+            if ignore_empty {
```

Comment:
Shouldn't this flag be inverted? To me, it seems like an error should be returned if the flag is _not_ activated.

To make sure it's working properly, could you add `[{a: 1} {a: 2} {a: 3}] | select -i b` to the tests?

Reply:
Thanks for pointing it out, kubouch. It should all work now.